quantum/quantum.h

```diff
@@ -261,6 +261,9 @@ void set_single_persistent_default_layer(uint8_t default_layer);
 
 uint16_t get_record_keycode(keyrecord_t *record, bool update_layer_cache);
 uint16_t get_event_keycode(keyevent_t event, bool update_layer_cache);
+bool     pre_process_record_quantum(keyrecord_t *record);
+bool     pre_process_record_quantum_kb(keyrecord_t *record);
+bool     pre_process_record_quantum_user(keyrecord_t *record);
```

Comment:
Should these functions be renamed to remove `quantum`? It would be more consistent with `process_record_user` and `post_process_record_user`.